ports/zephyr/moduos.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019, NXP
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "py/obj.h"
+
+#include "zfs.h"
+#include "zfs_file.h"
+
+STATIC const mp_rom_map_elem_t uos_module_globals_table[] = {
+    { MP_ROM_QSTR(MP_QSTR___name__), MP_ROM_QSTR(MP_QSTR_uos) },
+#ifdef CONFIG_FAT_FILESYSTEM_ELM
+    { MP_ROM_QSTR(MP_QSTR_ZfsFat), MP_ROM_PTR(&zfs_fat_type) },
+#endif
+#ifdef CONFIG_FILE_SYSTEM_LITTLEFS
+    { MP_ROM_QSTR(MP_QSTR_ZfsLittle), MP_ROM_PTR(&zfs_little_type) },
```

Comment:
Does it use littlefs v1 or v2?  Would be good to rename this type to match, either `ZfsLfs1` or `ZfsLfs2` (other ports name it `VfsLfs1` and `VfsLfs2`).

Reply:
v2